localized/ja/UserGuide-chapter1.adoc

```diff
@@ -0,0 +1,149 @@
+= Chapter 1. OptaPlanner イントロダクション
+:awestruct-description: Chapter 1. OptaPlanner イントロダクション
+:awestruct-layout: localizedBase
+:awestruct-lang: jp
+:awestruct-priority: 1.0
+:showtitle:
+xxx
+
+////
+
+Failed, to be deleted soon >>>>>
```

Comment:
CC @tkobayas
Thank you, Ishii-san. This is indeed a good start, a very nice step forward to the old situation (with any translations). 

German index.html fixed.